README.md

```diff
@@ -2,13 +2,25 @@
 
 ## About
 
-This repository has a series of examples of migrating your cloud.gov applications
+This repository has a series of examples for migrating your cloud.gov applications
 from using the `redis32` service running on the `standard-ha`, `standard`, and `micro` to the
```

Comment:
Wondering if it's useful to list out all of the plans (e.g., `standard-ha`, `standard`, and `micro`) since the example covers all plans, rather than a subset of them. Might just be clearer to say "migrating from the older `redis32` service to the new `redis` service.

Reply:
Totally.